app.js

```diff
@@ -0,0 +1,66 @@
+'use strict';
+
+module.exports = {
+  englishToPigLatin: function(word) {
+    //word verification
+    if (typeof word !== 'string' || Number(word) / 2) {
+      console.log('Please enter a valid word.');
+      return false;
+    }
+
+    //multi word verification
+    if (word.split('').indexOf(' ') !== -1) {
+      console.log('Please input a single word.');
+      return false;
+    }
+
+    let firstLetterVowel = 'AEIOUaeiou';
+    let appendage = 'ay';
+    let firstLetter = word.slice(0, 1);
+
+    if (firstLetterVowel.indexOf(firstLetter) === -1) {
+      let pigLatin = word
+        .split('')
+        .splice(1, word.length)
+        .join('')
+        .concat(`-${firstLetter}${appendage}`);
+      return pigLatin;
+    } else {
+      let pigLatin = word.concat(`-${appendage}`);
+      return pigLatin;
+    }
+  },
+
+  pigLatinToEnglish: function(word) {
+    //word verification
+    if (typeof word !== 'string' || Number(word) / 2) {
```

Comment:
Not sure what the second part of this if statement is trying to handle? It's turning the word into a number and dividing by 2? So basically as long as it's a number that's greater than 0?